controllers/chat/group/leaveGroupMember.js

```diff
@@ -0,0 +1,34 @@
+const BetterSocialCore = require('../../../services/bettersocial');
+const {ResponseSuccess} = require('../../../utils/Responses');
+const ErrorResponse = require('../../../utils/response/ErrorResponse');
+const Getstream = require('../../../vendor/getstream');
+
+const leaveGroupMember = async (req, res) => {
+  const {userId} = req;
+  const {channelId} = req.body;
+
+  try {
+    const response = await Getstream.chat.removeGroupMember(channelId, userId);
+    const {channel, channelApiResponse} = response.data || {};
+
+    let channelResponse;
+    try {
+      const {newChannelName, betterChannelMember, betterChannelMemberObject, updatedChannel} =
+        await BetterSocialCore.chat.updateBetterChannelMembers(channel, channelApiResponse, true);
+
+      channelResponse = updatedChannel;
+
+      channelApiResponse.channel.name = newChannelName;
+      channelApiResponse.channel.better_channel_member = betterChannelMember;
+      channelApiResponse.channel.better_channel_member_object = betterChannelMemberObject;
+    } catch (e) {
+      console.log(e);
+    }
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The error handling in this block is insufficient. It only logs the error and does not handle it properly. This could lead to unexpected behavior if an error occurs during the execution of `BetterSocialCore.chat.updateBetterChannelMembers`. Consider throwing the error or returning a response with an appropriate error message.

```diff
-      console.log(e);
+      throw e;
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->